lib/registrant.rb

```diff
@@ -0,0 +1,21 @@
+class Registrant
+  attr_accessor :name, :age, :permit, :license_data
```

Comment:
I'm curious why these are all attr_accessors. Rule of thumb: keep everything an attr_reader until you absolutely must have an attr_accessor. We want to limit the ability of outside classes to make changes to these attributes.